doc/source/indexing.rst

```diff
@@ -1163,25 +1167,12 @@ with the name ``a``.
 If instead you don't want to or cannot name your index, you can use the name
 ``index`` in your query expression:
 
-.. ipython:: python
-   :suppress:
-
-   old_index = index
-   del index
```

Comment:
do you mind rendering the page and making sure the result is not changed by removing these? can't really understand why these blocks are needed, but may be I'm missing something

Reply:
I removed the `:suppress:` which resulted in 
```
In [211]: old_index = index
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-211-27ea1dd9dc9d> in <module>
----> 1 old_index = index

NameError: name 'index' is not defined

In [212]: del index
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-212-b747d883ab88> in <module>
----> 1 del index

NameError: name 'index' is not defined
```
Same for the other (If the try block is removed as well). I doubt it makes any difference.

I noticed a `df.index` in the block above so I tried to replace it with that, but `del` is not defined.